### PR TITLE
Create issue_template.md

### LINKED ISSUE #654: Create GitHub issue template
It would be helpful to users, I think, to create an issue template that asks specifically for things like:

* platform, output of `nbgrader --version` and versions of other things (e.g. JupyterHub)
* summary of issue
* expected behavior
* actual behavior

See https://help.github.com/articles/creating-an-issue-template-for-your-repository/ for instructions on creating the template.

### PR DESCRIPTION
**Pull request by fm75**
Fixes #654

### CODE REVIEW

**Comment by fm75** [comment]
@jhamrick I just gave this a try since it was sprint-friendly.

File: docs/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Operating system
+
+### `nbgrader --version`
+
+### Notebook - Help, About version info
+
```
Comment (lgpage) [inline docs/issue_template.md]: Please can you include

    ### `jupyterhub --version` (if used with JupyterHub)

**Comment by fm75** [comment]
Addresses requested change

**Comment by fm75** [comment]
I don't have any access to the AppVeyor or Travis CI builds. The fact that it failed in python 2.7 seems silly since the changes have nothing to do with python at all. 

If you want to test this, put the same file in the same directory in some repo that you own, not in a forked repo. Push it to github, then try to add an issue. It worked as expected for me. One oddity is that if you do it a second time, without clearing cache/cookies (?), your second entry of issues will start out looking like your first.

**Comment by lgpage** [comment]
@fm75 the CI failure is unrelated to this. Please can you also include a JupyterHub version line, something like:

    ### `jupyterhub --version` (if used with JupyterHub)

Then I'll go ahead and merge this. Thanks.

**Comment by jzf2101** [comment]
@fm75 when you push your next changes please put `[ci skip]` in the commit message

**Comment by fm75** [comment]
Changes made.
[ ci skip ] added to commit comment

**Pull request merged**